ReactSkia/components/RSkComponentText.cpp

```diff
@@ -46,11 +46,10 @@ void RSkComponentParagraph::OnPaint(SkCanvas *canvas) {
   /*    - use parent paragraph builder to add text & push style */
   /*    - draw the paragraph, when we reach the last fragment attachment*/
   if(parent) {
-      parent->expectedAttachmentCount += textLayoutManager_.buildParagraph(
-                                                    data.attributedString,
-                                                    props.paragraphAttributes,
-                                                    true,
-                                                    parent->paraBuilder);
+      parent->expectedAttachmentCount += data.layoutManager->buildParagraph(data.attributedString,
```

Comment:
apologize for late review and having a question here. where is the `data.layoutManager` coming from? maybe some changes are missing from the pr?

Reply:
The layoutmanager handle is part of Paragraph State.The data value is read from state which is received in ShadowView. (Refer Line#40)

Reply:
got it. it's from react-native. thanks for your answer.